lib/components/form/date-time-modal.js

```diff
@@ -28,15 +28,15 @@ class DateTimeModal extends Component {
           <StyledDateTimeSelector
             className='date-time-selector'
             date={date}
+            dateFormatLegacy={dateFormatLegacy}
             departArrive={departArrive}
             onQueryParamChange={setQueryParam}
-            time={time}
             // These props below are for Safari on MacOS, and legacy browsers
             // that don't support `<input type="time|date">`.
             // These props are not relevant in modern browsers,
             // where `<input type="time|date">` already
             // formats the time|date according to the OS settings.
```

Comment:
Comment is now out-of-date since it appeared to also apply to `dateFormatLegacy`.

Reply:
Good catch, fixed in https://github.com/opentripplanner/otp-react-redux/pull/390/commits/1a526dd2886f67b837dfe118666f9d9d719988f5.